contentcuration/contentcuration/frontend/shared/views/form/__tests__/contentDefaults.spec.js

```diff
@@ -0,0 +1,172 @@
+import { mount } from '@vue/test-utils';
+import Vue from 'vue';
+import Vuetify from 'vuetify';
+import camelCase from 'lodash/camelCase';
+import ContentDefaults from '../ContentDefaults.vue';
+
+Vue.use(Vuetify);
+
+function makeWrapper(contentDefaults) {
+  return mount(ContentDefaults, {
+    sync: true,
```

Comment:
Why is `sync` needed here?

Reply:
Ah, good catch. I was having trouble with this particular test, and that flag was something I tested out to see if it would resolve the issue. I think it'll be safe to remove now.